### ISSUE TITLE
Can't set default country

### ISSUE THREAD

**Issue by imsam67** [catamphetamine/react-phone-number-input#104]
When I try to set the default country, I'm getting this error:
![capture](https://user-images.githubusercontent.com/15844030/35774791-45c4ef70-0948-11e8-969e-8401e0a931be.PNG)

I'm setting the country as below:
`<Phone country="US" value={this.state.myValue} onChange={e => this.myFunction(e)} />`

Any idea what the issue is?

**Comment by catamphetamine** [comment]
I didn't encounter such a bug but you can debug it yourself by editing `node_modules/libphonenumber-js/.../parse.js` at line 510 by inserting `console.log(text, typeof text)` there.
`...` folder is either `es6` or `build`.

https://unpkg.com/libphonenumber-js@1.0.8/es6/parse.js

```js
export function extract_formatted_phone_number(text) {
	if (!text || text.length > MAX_INPUT_STRING_LENGTH) {
		return '';
	}

	// Attempt to extract a possible number from the string passed in

	var starts_at = text.search(PHONE_NUMBER_START_PATTERN);

	if (starts_at < 0) {
		return '';
	}

	return text
	// Trim everything to the left of the phone number
	.slice(starts_at)
	// Remove trailing non-numerical characters
	.replace(AFTER_PHONE_NUMBER_END_PATTERN, '');
}
```

**Comment by imsam67** [comment]
Thank you for your response. Interestingly, the problem was caused by something else. In my app, I have a phone number object, not a string because I break down the phone number into components e.g. country, area code, phone number, extension, etc. So I was binding that object to your component for its value but your component expects a simple string. That was the cause of this issue. I was seeing the error in the console but I was trying to fix this country selection issue first. When I fixed the value error caused by binding an object, the whole thing started to work fine. Thanks again!
